api/src/modules/document-drive/drives-resolver.ts

```diff
@@ -113,3 +120,20 @@ export const deleteDrive = mutationField('deleteDrive', {
     return true;
   },
 });
+
+export const setDriveIcon = mutationField('setDriveIcon', {
+  type: 'Boolean',
+  args: {
+    id: nonNull('String'),
+    icon: nonNull('String'),
+  },
+  resolve: async (_parent, { id, icon }, ctx: Context) => {
+    try {
+      await ctx.prisma.document.setDriveIcon(id, icon);
```

Comment:
the status might not be success, we need to check and throw the associated error